miqcli/collections/actions.py

```diff
@@ -14,8 +14,10 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
+from miqcli.cli.collection import Collection
 
-class Collections(object):
+
+class Collections(Collection):
```

Comment:
In order to maintain a clear separation of concerns between the "library" components and the CLI, at no point should any module in the `miqcli.collections` namespace depend on anything from the `miqcli.cli` namespace. I think it's a great idea to have a `Collection` base class, but that class should probably be defined in `miqcli.collections.base` or some namespace along those lines.

Reply:
The more I think about this, I agree with @seandst on this suggestion. I was back and forth on the location with the class also. Having this separation will maintain a clean code base structure.

Reply:
Thanks for the reviews, all the comments made sense, I have made the suggested updates.